pillarbox-monitoring-terraform/20-pillarbox-monitoring-app/data_transfer_service.tf

```diff
@@ -0,0 +1,119 @@
+# -----------------------------------
+# Security Group for Data Transfer Task
+# -----------------------------------
+
+resource "aws_security_group" "transfer_task_sg" {
+  name   = "data-transfer-task-sg"
+  vpc_id = data.aws_vpc.main_vpc.id
+
+  # Egress rule allowing all outbound traffic
+  egress {
+    description = "Allow all outbound traffic"
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags = {
+    Name = "data-transfer-task-sg"
+  }
+}
+
+# -----------------------------------
+# CloudWatch Log Group for Data Transfer Task
+# -----------------------------------
+
+resource "aws_cloudwatch_log_group" "transfer_log_group" {
+  name              = "/ecs/pillarbox-monitoring-transfer"
+  retention_in_days = 7 # Adjust the retention period as necessary
+
+  tags = {
+    Name = "data-transfer-log-group"
+  }
+}
+
+# -----------------------------------
+# ECS Task Definition for Data Transfer Service
+# -----------------------------------
+
+resource "aws_ecs_task_definition" "transfer_task" {
+  family                   = "pillarbox-monitoring-transfer"
+  network_mode             = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  cpu                      = local.transfer_task.cpu
+  memory                   = local.transfer_task.memory
+  execution_role_arn       = aws_iam_role.ecs_task_execution_role.arn
+  task_role_arn            = aws_iam_role.ecs_task_role.arn
+
+  # Define the container and its configuration
+  container_definitions = jsonencode([
+    {
+      name                   = "pillarbox-monitoring-transfer"
+      image                  = "${local.ecr_repository}/pillarbox-monitoring-transfer:latest"
+      readonlyRootFilesystem = true
+
+      # Port mapping for the container
+      portMappings = [
+        {
+          hostPort      = 8081
+          containerPort = 8081
+          protocol      = "tcp"
+        }
+      ]
+
+      # Environment variables for connecting to SSE and OpenSearch services
+      environment = [
+        {
+          name  = "PILLARBOX_QOS_SSE_URI"
```

Comment:
```suggestion
          name  = "PILLARBOX_MONITORING_SSE_URI"
```

Reply:
I like the suggestion but unfortunately this one is tied to a configuration property for the transfer service: https://github.com/SRGSSR/pillarbox-monitoring-transfer/blob/089f51c828bc460491c1a3f185c23fa40c34427e/src/main/resources/application.yml#L5